builders/ethereum/dev-env/hardhat.md

```diff
@@ -62,19 +62,19 @@ You will need to create a Hardhat project if you don't already have one. You can
 
 5. You'll be prompted with a series of questions to set up your project:
 
-    - Choose **Hardhat 3 Beta (recommended for new projects)** rather than Hardhat 2
-    - Choose where to initialize the project (default is current directory)
-    - Confirm converting to ESM (required for Hardhat v3)
+    - Choose **Hardhat 3 Beta (recommended for new projects)** rather than Hardhat 2.
+    - Choose where to initialize the project (default is current directory).
+    - Confirm converting to ESM (required for Hardhat v3).
     - Select the type of project to initialize:
-        - A TypeScript Hardhat project using Node Test Runner and Viem
-        - A TypeScript Hardhat project using Mocha and Ethers.js
+        - A TypeScript Hardhat project using Node Test Runner and Viem.
+        - A TypeScript Hardhat project using Mocha and Ethers.js.
 
     For this example, you can choose either option based on your preference. If you choose the Mocha and Ethers.js option, you'll get a project structure with:
     
-    - A sample contract in `contracts/Counter.sol`
-    - A test file in `test/Counter.ts`
-    - TypeScript configuration
-    - Mocha and Ethers.js dependencies
+    - A sample contract in `contracts/Counter.sol`.
+    - A test file in `test/Counter.ts`.
+    - TypeScript configuration.
+    - Mocha and Ethers.js dependencies.
```

Comment:
No punctuation needed in this list as there are no verbs in the list items. Check the GDDSG for reference: https://developers.google.com/style/lists#numbered-lettered-bulleted-lists